src/main/kotlin/graphics/scenery/PeriodicTable.kt

```diff
@@ -37,7 +40,7 @@ open class PeriodicTable {
     init {
         //parsing the json file
         val mapper = jacksonObjectMapper()
-        val file = FileReader("src/main/resources/graphics/scenery/proteins/PubChemElements_all.json").readText()
+        val file = this::class.java.getResource("src/main/resources/graphics/scenery/proteins/PubChemElements_all.json").readText()
```

Comment:
That again is not going to work when deployed as a JAR: The paths given to `getResource` are always relative to the class it's being called on. So if your `PubChemElements_all.json` is in the same package as the `PeriodicTable` class, all you need there is the file name 👍

Reply:
Strangely enough it still doesn't work when I put both in the same package. getResource() still throws a NullPointerException